File: pgm/ABC_Java/ABC_062_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        final int one = 1;
        final int two = 2;
        final int three = 3;
        final int four = 4;
        final int five = 5;
        final int six = 6;
        final int seven = 7;
        final int eight = 8;
        final int nine = 9;
        final int ten = 10;
        final int eleven = 11;
        final int twelve = 12;
        final int zero = 0;
        final Set<Integer> g1 =
            Collections.unmodifiableSet (new HashSet<Integer> (
                Arrays.asList (one, three, five, seven, eight, ten, twelve)));
        final Set<Integer> g2 = Collections.unmodifiableSet (
            new HashSet<Integer> (Arrays.asList (four, six, nine, eleven)));
        final Set<Integer> g3 = Collections.unmodifiableSet (
            new HashSet<Integer> (Arrays.asList (two)));
        final int x = Integer.parseInt (arr[zero]);
        final int y = Integer.parseInt (arr[one]);
        System.out.println ((g1.contains (x) && g1.contains (y)) ||
                                    (g2.contains (x) && g2.contains (y)) ||
                                    (g3.contains (x) && g3.contains (y))
                                ? Yes
                                : No);
        System.exit (0);
    }
}

File: pgm/ABC_R/ABC_062_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
Yes <- "Yes"
No <- "No"
blank <- ""
space <- " "
one <- 1L
two <- 2L
three <- 3L
four <- 4L
five <- 5L
six <- 6L
seven <- 7L
eight <- 8L
nine <- 9L
ten <- 10L
eleven <- 11L
twelve <- 12L
x <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][one] |>
    as.integer()
y <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][two] |>
    as.integer()
g1 <- c(one, three, five, seven, eight, ten, twelve)
g2 <- c(four, six, nine, eleven)
g3 <- c(two)
if (x %in% g1 && y %in% g1 ||
        x %in% g2 && y %in% g2 ||
        x %in% g3 && y %in% g3) {
    Yes |>
        cat("\n", sep = blank)
} else {
    No |>
        cat("\n", sep = blank)
}
q("no")